controllers/ldap.go

```diff
@@ -422,6 +422,10 @@ func (c *ApiController) SyncLdapUsers() {
 		c.ResponseError(err.Error())
 		return
 	}
+
+	mappingRb := object.NewRecordBuilderFromCtx(c.Ctx)
+	mappingCtx := context.WithValue(goCtx, object.RoleMappingRecordDataKey, mappingRb)
```

Comment:
If you dont need to separately store mappingCtx and goCtx, you can rewrite goCtx value with result of context.WithValue(), resulting in
`
goCtx := context.WithValue(goCtx, object.RoleMappingRecordDataKey, mappingRb)
`
This makes current scope less cluttered with not needed variables.

Reply:
done